ChallengesWithTestsMarkVII/ChallengesSet01.cs

```diff
@@ -8,37 +8,50 @@ public class ChallengesSet01
     {
         public bool AreTwoNumbersTheSame(int num1, int num2)
         {
-            throw new NotImplementedException();
+            return (num1==num2);
         }
 
         public double Subtract(double minuend, double subtrahend)
         {
-            throw new NotImplementedException();
+            return(minuend-subtrahend);
         }
 
         public int Add(int number1, int number2)
         {
-            throw new NotImplementedException();
+            return (number1 + number2);
         }
 
         public int GetSmallestNumber(int number1, int number2)
         {
-            throw new NotImplementedException();
+            if (number1 < number2)
```

Comment:
This is good and works; however, there's a `Math.Min()` method that would make this simpler/shorter.